apps/src/p5lab/spritelab/blocks.js

```diff
@@ -210,14 +216,25 @@ const customInputTypes = {
         getStore().getState().pageConstants &&
         getStore().getState().pageConstants.showAnimationMode
       ) {
-        buttons = [
-          {
-            text: i18n.more(),
-            action: () => {
-              getStore().dispatch(showBackground(Goal.NEW_ANIMATION));
-            }
-          }
-        ];
+        buttons = experiments.isEnabled('backgroundsTab')
+          ? [
+              {
+                text: i18n.backgroundMode(),
```

Comment:
this button is overflowing a bit now that the text is longer, can we increase the width of the dropdown or decrease the font size?
![Screenshot 2023-02-02 at 2 46 41 PM](https://user-images.githubusercontent.com/33666587/216469134-7f3a48ba-db41-4ce8-b189-0c25ba2bfa6c.png)

Reply:
Strange! That should have been fixed by this: https://github.com/code-dot-org/blockly/pull/308
Maybe I don't understand the process for getting changes from blockly fork to actually get built...

Reply:
do we need to publish a new npm version, then start using that new version?

Reply:
Ah, I'm seeing instructions that I had missed here: https://github.com/code-dot-org/blockly#publishing-changes

I'll look into this!